R/Sound_DMR.R

```diff
@@ -748,30 +745,29 @@ individual_DMR <- function(Output_Frame, ZoomFrame_filtered, experimental_design
 #' @return Output_Frame (df)
 #' @export
 
-find_DMR <- function(Output_Frame, dmr_obj, fixed = c('Group'),
-                     random = c('Plant'), reads_threshold = 3,
-                     model, control = '', analysis_type) {
+find_DMR<- function(ms, dmr_obj, fixed = c('Group'),
```

Comment:
Since this function inherits param doc string from `individual_DMR` i think you should use `methylsummary` instead of `ms`